### ISSUE TITLE
WebCodecs AudioEncoder bitrateMode

### ISSUE THREAD

**Issue by tguilbert-google** [mozilla/standards-positions#837]
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: WebCodecs AudioEncoder bitrateMode
* Specification or proposal URL (if available): https://w3c.github.io/webcodecs/#dom-audioencoderconfig-bitratemode
* Explainer URL (if available): 
* Caniuse.com URL (optional): 
* Bugzilla URL (optional): 
* Mozillians who can provide input (optional): @padenot 

### Other information

The Media WG has agreed to add a `bitrateMode` to `AudioEncoderConfig`, merged into the spec as part of https://github.com/w3c/webcodecs/pull/657.

This flag will allow web authors to configure WebCodecs AudioEncoders to use a variable bitrate or a constant bitrate. Specific encoder implementations might have slightly different terminology (e.g. `CBR` vs `VBR` for Opus), but all of them should map to the general concept of `"constant"` versus `"variable"` bitrate.

Work on the Chromium side is being [tracked here](https://chromestatus.com/feature/5084320053592064).

**Comment by tguilbert-google** [comment]
This request for a standards position is following the Blink feature launch process; it is perhaps heavy-handed for a change of this size (adding a single entry in a dictionary, approved by the Media WG).

Are there any comments as to whether this type of change should go through the standards position request?

**Comment by padenot** [comment]
I don't think this warrants an entry here, considering we're active participants of the group, haven't voiced any opposition on this particular feature, and haven't released Web Codecs, that we are currently implementing (see https://github.com/mozilla/standards-positions/issues/209). 

Please note that there hasn't been a review by any editor of the specification before merging this PR however, and there's been feedback from a single implementer (Chromium). This isn't how specification work is supposed to happen.

**Comment by padenot** [comment]
I'd like to make it clear that I agree with this particular spec change, though, so this instance isn't a problem in practice, and that the overwhelming majority of Web Codecs issues and PRs are handled appropriately.

I trust that if that it had been something more substantial there would have been more discussions, but flagging implementors and editors for review is easy to do, and we get notifications -- worst case you can ping us if we seem to not take care of a request quickly enough.

**Comment by chrisn** [comment]
That's true, and my comment is as much a reminder to myself as anyone else.

**Comment by tguilbert-google** [comment]
My apologies: I went back and checked the Media WG meeting notes, and that change was not brought up in the WG. It's unclear whether this was brought up in the editors call as well; probably not, since there is no note.

It seems like we did merge that change too fast. We will double check and make sure to CC spec editors on all future changes.

Sorry about that, and thank you for having taken a look at this current issue.
